test/e2e/components/vl-form-validation.js

```diff
@@ -1,6 +1,32 @@
 const { VlElement } = require('vl-ui-core').Test;
 
+
 class VlFormValidation extends VlElement {  
+   
+    async getErrorId() {
+        return this.getAttribute('data-vl-error-id');
+    }
+
+    static async elementHasisDataRequired(element) {
+        return element.hasAttribute('data-required');
+    }
+
+    static async getErrorPlaceholder(element) {
+        return element.getAttribute('data-vl-error-placeholder');
+    }
+
+    async toontFoutmeldingenVoorElement(element) {
+        const errorId = await this.getErrorId();
+        const errorPlaceholderIdFromElement = await VlFormValidation.getErrorPlaceholder(element);
+        return errorId == errorPlaceholderIdFromElement;
+    }
+
+    async getErrorMessage() {
+        return this.getText();
+    }
+  
 }
 
+
+
```

Comment:
Ik denk dat VlFormValidation gewoon (zoals bij de source code) een base class moet zijn waarvan je de attributen gewoon kunt 'getten', maar dat een deel van deze logica en/of testen dan in VlFormValidationMessage thuis hoort?
Dus mogelijks hier geen testen nodig, maar te schrijven bij VlFormValidationMessage?

Reply:
Ja, voorlopig hebben we afgesproken om de testen hier te behouden. Ze zullen eventueel later naar de gepaste locatie gekopieerd worden.